builtin/docker/platform.go

```diff
@@ -349,12 +452,12 @@ func (p *Platform) pullImage(cli *client.Client, log hclog.Logger, ui terminal.U
 
 		// if we have images do not pull
 		if len(sum) > 0 {
+			s.Done()
```

Comment:
I _think_ Evans recent work with stepgroups is going to fix this, but I noticed  this line never resolves as Done:

![Screenshot from 2021-05-18 07-50-02](https://user-images.githubusercontent.com/810277/118673194-aaefce80-b7ad-11eb-88a1-eecb5c0ef7f2.png)

Reply:
We could add an update here that's like:

```suggestion
			s.Update("Docker image %q up to date!", in)
			s.Done()
```